addons/website/tests/test_grid_layout.py

```diff
@@ -29,3 +30,7 @@ def test_01_replace_grid_image(self):
             'datas': base64.b64encode(req.content),
         })
         self.start_tour(self.env['website'].get_client_action_url('/'), 'website_replace_grid_image', login="admin")
+
+    def test_02_scroll_to_new_grid_item(self):
+        create_image_attachment(self.env, '/web/image/website.s_banner_default_image', 's_banner_default_image.jpg')
```

Comment:
Can't we use your new util somewhere else ? :thinking:

Reply:
But then I should make a dedicated commit for that right ? If I have to change that everywhere we create an image :thinking:

Reply:
Yes you can create a separate commit, before the scroll commit. Create the util and use it everywhere you can (in website) :+1: .